addons/web_editor/views/editor.xml

```diff
@@ -131,6 +131,7 @@
         <script type="text/javascript" src="/web_editor/static/lib/vkbeautify/vkbeautify.0.99.00.beta.js"/>
         <script type="text/javascript" src="/web_editor/static/src/js/common/ace.js"/>
         <script type="text/javascript" src="/web_editor/static/src/js/wysiwyg/root.js"/>
+        <script type="text/javascript" src="/web_editor/static/src/js/common/browser_compatibility.js"/>
```

Comment:
Actually, in stable versions, you cannot change the XML structure unless in some specific cases like adding a class on a node for a non-critical bugs. This is because:
- Custo might xpath the XML you want to modify, so changing it could break those xpath (not really important here for adding an asset file)
- Users won't receive the fix unless they perform a -u of the web_editor app (that's important here: we want users to receive the fix)

In 15.0, there is a new system of assets which allows to add files in stable. In 13.0, we'll have to find an existing JS file where it makes sense to add this. I'll do it myself so you can move on to something else :+1: Actually, I imagine it would make sense to put this in the /web/ app directly... but the framework team would probably not agree and it's not really important as the web_editor app is automatically installed with the web app anyway :)